modules/config/URLProcessor.js

```diff
@@ -1,69 +1,92 @@
-/* global config, getConfigParamsFromUrl, interfaceConfig, loggingConfig */
+/* global config, interfaceConfig, loggingConfig */
 
 const logger = require("jitsi-meet-logger").getLogger(__filename);
 
-var configUtils = require('./Util');
-var params = {};
+import { getConfigParamsFromUrl } from '../../react/features/base/config';
 
-if (typeof getConfigParamsFromUrl === 'function') {
-    params = getConfigParamsFromUrl();
-}
+import configUtils from './Util';
 
-var URLProcessor = {
-    setConfigParametersFromUrl: function () {
-        // Convert 'params' to JSON object
-        // We have:
-        // {
-        //   "config.disableAudioLevels": false,
-        //   "config.channelLastN": -1,
-        //   "interfaceConfig.APP_NAME": "Jitsi Meet"
-        // }
-        // We want to have:
-        // {
-        //   "config": {
-        //     "disableAudioLevels": false,
-        //     "channelLastN": -1
-        //   },
-        //   interfaceConfig: {
-        //     APP_NAME: "Jitsi Meet"
-        //   }
-        // }
-        var configJSON = {
-            config: {},
-            interfaceConfig: {},
-            loggingConfig: {}
-        };
-        for (var key in params) {
-            if (typeof key !== "string") {
-                logger.warn("Invalid config key: ", key);
-                continue;
-            }
-            var confObj = null, confKey;
-            if (key.indexOf("config.") === 0) {
-                confObj = configJSON.config;
-                confKey = key.substr("config.".length);
+/**
+ *  URL params with this prefix should be merged to config.
+ */
+const CONFIG_PREFIX = 'config.';
+
+/**
+ *  URL params with this prefix should be merged to interface config.
+ */
+const INTERFACE_CONFIG_PREFIX = 'interfaceConfig.';
+
+/**
+ *  URL params with this prefix should be merged to logging config.
+ */
+const LOGGING_CONFIG_PREFIX = 'loggingConfig.';
+
+/**
+ * Config keys to be ignored.
+ *
+ * @type Set
+ */
+const KEYS_TO_IGNORE = new Set([
+    'analyticsScriptUrls',
+    'callStatsCustomScriptUrl'
+]);
+
+/**
+ * Convert 'URL_PARAMS' to JSON object
+ * We have:
+ * {
+ *      "config.disableAudioLevels": false,
+ *      "config.channelLastN": -1,
+ *      "interfaceConfig.APP_NAME": "Jitsi Meet"
+ * }
+ * We want to have:
+ * {
+ *      "config": {
+ *          "disableAudioLevels": false,
+ *          "channelLastN": -1
+ *      },
+ *      interfaceConfig: {
+ *          "APP_NAME": "Jitsi Meet"
+ *      }
+ * }
+ */
+export function setConfigParametersFromUrl() {
```

Comment:
I'm not sure why modules/config/URLProcessor.js continues to exist. It's used in exactly one location that's inside the react/ directory, I asked for the useless URLProcessor class to be removed and you complied. Unless you come back with a reason why the one and only function `setConfigParametersFromUrl` needs to remain in a file which doesn't bear its name, I'm going to kill modules/config/URLProcessor.js.